specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/2015-11-01-preview/OperationsManagement.json

```diff
@@ -0,0 +1,422 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Azure Log Analytics - Operations Management",
+        "description": "Azure Log Analytics API reference for Solution.",
+        "version": "2015-11-01-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "security": [
+        {
+            "azure_auth": [
+                "user_impersonation"
+            ]
+        }
+    ],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {       
+        "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.OperationsManagement/solutions/{solutionName}": {
+            "put": {
+                "tags": [
+                    "Solution"
+                ],
+                "x-ms-examples": {
+                     "SolutionCreate": { "$ref": "./examples/SolutionCreate.json" }
+                },
+                "operationId": "OperationsManagement_CreateSolution",
```

Comment:
I think this should be "Solution_CreateOrUpdate", @ravbhatnagar would you agree?

Reply:
Solution_CreateOrUpdate or Solutions_CreateOrUpdate

Reply:
Solutions_CreateOrUpdate